### PR TITLE
Add an API end-point that lists the available resources and their UID

### PR DESCRIPTION
**Pull request by mprahl**
This will be used for the initial landing page on the frontend.

### CODE REVIEW

File: purview/api/v1.py [hunk 1]
```diff
@@ -93,3 +93,19 @@ def get_resource_story(resource, uid):
...
+@api_v1.route('/resources')
+def get_resources():
+    """
+    List the available resources and their unique ID property.
+
+    :return: a Flask JSON response
+    :rtype: flask.Response
+    """
+    resources = {}
+    for model in all_models:
```
Comment (yashvardhannanavati) [inline purview/api/v1.py]: It might be easier to iterate through the story_flow dict instead of all_models

Comment (yashvardhannanavati) [reply purview/api/v1.py]: since story_flow already has the uid of a resource as an attribute

**Comment by mprahl** [comment]
@yashvardhannanavati fixed. Thanks for the review!

**Comment by yashvardhannanavati** [comment]
There seems to be an unused import here. Can you please fix this

**Comment by mprahl** [comment]
@yashvardhannanavati I noticed that a few mins ago and fixed it.

**Comment by yashvardhannanavati** [comment]
:+1:

**Pull request merged**